sourcemash/frontend/static/scss/sourcemash.scss

```diff
@@ -32,6 +32,27 @@
   }
 }
 
+.help-fixed-action-btn {
```

Comment:
Really nice! Well, mobile view looks crappy but the button looks good haha.

![Uploading Screen Shot 2015-05-20 at 5.42.25 PM.png…]()